routes/questions.js

```diff
@@ -0,0 +1,19 @@
+const router = require('express').Router()
+
+const questionController = require('~/controllers/question')
+const asyncWrapper = require('~/middlewares/asyncWrapper')
+const { authMiddleware, restrictTo } = require('~/middlewares/auth')
+const isEntityValid = require('~/middlewares/entityValidation')
+const Question = require('~/models/question')
+const {
+  roles: { TUTOR }
+} = require('~/consts/auth')
+
+const body = [{ model: Question, idName: 'question' }]
```

Comment:
there is no question field in body to validate

Reply:
Fixed